src/garrage/ConsoleDialogue.java

```diff
@@ -0,0 +1,239 @@
+            }
+            String answer = scanner.nextLine();
+            switch (answer) {
+                case ("e"):
+                case ("E"):
+                    generalDialogue = false;
+                case ("d"):
+                case ("D"):
+                    System.out.println(result.toString());
+                    break;
+                case ("m"):
+                case ("M"):
+                    modifyCarInter(result);
+                    break;
+                default:
+                    System.err.println("Input \"m\" or \"d\" or \"e\"");
+                    break;
+            }
+        }
+        return null;
+    }
+
+    private Car createCarConsole() {
+        Car result;
+        String creationDate;
+        String engineType = "Petrol";
+        int maxSpeed = 200;
+        int timeTillHundred = 9;
+        int passengersQuantity = 5;
+        int passengersInside = 0;
+        int currentSpeed = 0;
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter date of garrage.Car creation, format is \"dd-MM-YY\"");
+        creationDate = scanner.nextLine();
+        System.out.println("Enter type of the engine Petrol/Disel");
+        engineType = scanner.nextLine();
+        System.out.println("Enter max speed of the garrage.Car (number below 500)");
+        maxSpeed = scanner.nextInt();
+        System.out.println("Enter time need to get 100 km/h (number below 50)");
+        timeTillHundred = scanner.nextInt();
+        System.out.println("Enter passengers capacity (number below 20)");
+        passengersQuantity = scanner.nextInt();
+        System.out.println("Enter number of passengers inside the car at the moment (number below "
+                + passengersQuantity + ")");
+        passengersInside = scanner.nextInt();
+        result = new Car(creationDate, engineType, maxSpeed, timeTillHundred, passengersQuantity, passengersInside,
+                currentSpeed);
+        System.out.println("garrage.Car is created!");
+
+        return result;
+    }
+
+
+    private Car modifyCarInter(Car car) {
+
+        boolean modifyDialogue = true;
+        Scanner scanner = new Scanner(System.in);
+        String answer;
+        System.out.println("You are in modify menu. Chose what do you want to change");
+        System.out.println("Change current speed - \"c\"");
+        System.out.println("Add passenger - \"a\"");
+        System.out.println("Remove passenger - \"r\"");
+        System.out.println("Modify door by index - \"md\"");
+        System.out.println("Modify wheel by index - \"mw\"");
+        System.out.println("Remove all the wheels - \"wr\"");
+        System.out.println("Add wheels - \"wa\"");
+        System.out.println("Exit to previous menu - \"e\"");
+
+        while (modifyDialogue) {
+            answer = scanner.nextLine();
+            switch (answer) {
+                case ("e"):
```

Comment:
You could use answer.toUpperCase(). And then you don't need case ("e"):
The case for other code.